docs/source/getting-started/basic-usage.rst

```diff
@@ -0,0 +1,61 @@
+
+In the center of the ViRelAy UI, the embedding viewer is situated. The embedding viewer displays the samples of the currently selected sample category. The samples are displayed as a point cloud using the currently selected embedding method. Each analysis in a project can contain multiple embedding methods. The colors of the sample points correspond to the cluster they belong to, according to the currently selected clustering.
+
+.. figure:: ../../images/getting-started/basic-usage/embedding-viewer.png
+    :alt: Embedding Viewer in the ViRelAy UI
+    :align: center
+
+    Figure 4: The embedding viewer in the ViRelAy UI, where the samples of the currently selected category are visualized using the selected embedding method.
+
+The embedding viewer supports panning and zooming. To start panning click and hold the right mouse button and start moving the mouse pointer. Scrolling the mouse wheel will zoom into the region the mouse pointer is currently pointing at. Alternatively, click and hold the middle mouse button to start zooming and move the mouse pointer up and down to zoom in and out. When hovering the mouse pointer over a sample point, the dataset sample, which is the input of the classifier, is previewed at the top left of the embedding viewer. Samples can be selected by clicking and holding the left mouse button and moving the mouse pointer to draw a selection rectangle. Sample points outside of the selection will be in a muted color, while the selected sample points will have an increased saturation.
+
+When sample points have been selected, the sample viewer at the bottom of the ViRelAy UI shows the samples corresponding to the selected sample points. Depending on the currently selected visualization mode, the sample viewer will display the dataset sample, which is the input to the classifier, the attribution of the classification, or an attribution superimposed onto the dataset sample. Attributions will be displayed as heatmaps using the currently selected color map. Both the visualization mode and the color map can be selected in the toolbox, which is situated right above the embedding viewer.
+
+On the far right of the ViRelAy UI, the eigenvalues of the embedding as well as the clusters of the currently selected clustering method can be seen. The eigenvalue plot shows the eigenvalues of the eigen decomposition of the currently selected embedding method in ascending order. Below the eigenvalue plot, a list of all clusters of the current clustering are displayed. These can be used to quickly select all sample points belonging to a cluster, by clicking the corresponding clustering.
+
+Finally, at the top of the ViRelAy UI, below the project selection, the toolbox is situated. The toolbox allows the user to navigate the data of the current project. The toolbox can be seen in Figure 5.
+
+.. figure:: ../../images/getting-started/basic-usage/toolbox.png
+    :alt: Toolbox in the ViRelAy UI
+    :align: center
```

Comment:
The `sphinxcontrib-images` extension only works for images inserted using the `image` directive, but we are using the `figure` directive, because it allows us to have an image caption. I haven't found an alternative with support for the `figure` directive. We have 3 options:

1. Leave it as it is
2. Use the `image` directive and remove the image captions
3. Somehow crop the images to make them smaller

Reply:
Maybe split them after the last drop-down and line them up vertically?

Reply:
I agree, this is the only image where it really is a problem, I am going to break it into smaller pieces and stack it vertically. Tell me if that is to your liking.